expr/ztests/cut-union.yaml

```diff
@@ -0,0 +1,17 @@
+zql: a=union(cut(x)),b=union(cut(x,s))
```

Comment:
I'm surprised this works without `put` in front of it. Should it?

Reply:
As we discussed, this is the aggregation rather than expression form.  More on this topic coming soon on the search/expr syntax PR.